<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceEnter</class>
 <widget class="QMainWindow" name="FaceEnter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
#cameraArea{
	border:1px solid black;
	border-radius: 2px
}</string>
   </property>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>住户姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_num">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>门 牌 号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_phone">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>住户电话：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_num">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enter">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始录入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_notice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>411</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注意事项
    请点击“开始录入”后，再面对设备。
    采集完成，点击“结束录入后，设备不再采集。”</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_phone">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_submit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>350</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_submit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>多角度采集</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
